LICENSE

```diff
@@ -0,0 +1,662 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
This is a pretty standard file to add to an open-source project.
There is a sweet explanation about this here at source hut:
https://man.sr.ht/license.md